modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JavaClientCodegen.java

```diff
@@ -47,6 +47,7 @@
 
     public JavaClientCodegen() {
         super();
+        supportsInheritance = true;
```

Comment:
This cannot be done at the moment. See : https://github.com/swagger-api/swagger-codegen/pull/1120#issuecomment-219290749
Does it work if you keep supportsInheritance to false ?